scripts/exregional_process_bufr.sh

```diff
@@ -238,7 +248,7 @@ fi
 #
 #-----------------------------------------------------------------------
 #
-if [ $run_lightning ]; then
+if [[ "$run_lightning" == true ]]; then
```

Comment:
Aren't these the doing the same thing?

Reply:
Either use
`if [[ "$run_lightning" == true ]]; then
`
or use
`if  $run_lightning then`
But `if [ $run_lightning ]; then` is not right.

Reply:
Yes, this is a bug fix. Someone thinks "if [[ "$run_lightning" == true ]]; then " is better way to go. Seems it is more clear.